### PR TITLE
SHow and hide mobile menu added

### PR DESCRIPTION
**Pull request by Oscar-Espinoza**
In this commit, I added the functionality for the menu to appear and disappear when you click on the toggle button or the close menu button

### CODE REVIEW

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Invalid Code Review Request ❌

Hi @Oscar-Espinoza, This is @sja-thedude your code reviewer.

While you made a great effort in this project, I cannot proceed to review your code.

You have submitted a project, that is a pair programming project and doesn't have commits from both partners. Also, all linters are not passing, just disable linter to fix it.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: app.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+document.querySelectorAll('.menu-btn').forEach((item) => {
+  item.addEventListener('click', () => {
+    const menu = document.getElementById('menu-mobile');
+    if (menu.classList.contains('active')) {
+      menu.classList.remove('active');
+    } else {
+      menu.classList.add('active');
+    }
+  });
```
Comment (Ugboaja-Uchechi) [inline app.js:9]: - [x] Great job with your design and functionalities 👏 . As stated in the project requirements, kindly make sure when a user clicks any of the mobile menu options, the correct part of the page is displayed. 🙏 
- [x] As stated in the project requirements, kindly make sure when a user clicks any of the mobile menu options, the mobile menu disappears. 🙏

**Review by Ugboaja-Uchechi — CHANGES_REQUESTED** [review]
Hi @Oscar-Espinoza, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- Linters are passing ✔️ 
- README is descriptive 👍 
- I love the last commit message 👏 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Olamarx — APPROVED** [review]
Hi @Oscar-Espinoza, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- All linters passed ✔️ 
- Mobile menu functionalities ✅ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**